src/App.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import './App.css';
+import data from './data/resume.json';
+
+function BuildBasicsElement(basics) {
+  /*
+    todo: Elements in basic info remaining below.
+    For profiles, would have considered creating a for loop
+    to get through each profile.
+    "profiles": [{
+      "network": "Twitter",
+      "username": "john",
+      "url": "http://twitter.com/john"
+    }]
+  */
+  const { location } = basics;
+  return (
+    <div className="App" align="left">
+      <h1>{basics.name}</h1>
+      <h3>{basics.label}</h3>
+      <h5>
+        {basics.location.address} {location.postalCode} {location.city} {location.region} {location.country}
+        <br></br>
+        {basics.phone}
+        <br></br>
+        {basics.email}
+        <br></br>
+        {basics.website}</h5>
+      <hr></hr>
+      <p className="header-summary">{basics.summary}</p>
+    </div>
+  );
+}
+
+function BuildWorkElement(work) {
```

Comment:
For component names, I think it would make more sense to take out the "Build" from the name to better match how React names their components (based off their documentation).